FreydCategoriesForCAP/gap/CategoryOfRows.gi

```diff
@@ -1235,6 +1235,36 @@ InstallMethod( Display,
     
 end );
 
+##
+InstallMethod( LaTeXOutput,
+               [ IsCategoryOfRowsObject ],
+               
+  function( obj )
+    
+    return Concatenation( "R^{1 \\times ", String( RankOfObject( obj ) ), "}" );
```

Comment:
I would have done this:

C := CapCategory( obj );

if IsBound( C!.ring_latex_output ) then
   R := C!.ring_latex_output;
else
   R := "R";
fi;
return Concatenation( "{", R, "}^{1 \\times ", String( RankOfObject( obj ) ), "}" );

In case one works over a field or the ring of integer and with more than one ring in the same session.

Reply:
What is `ring_latex_output`?

Reply:
The name of the ring in latex-code: for example "\\mathbb{Z}", or  "\\mathbb{Q}"

Reply:
I think the appropriate place for such LaTeX code would be in `RingsForHomalg`, preferably callable by `LaTeXStringOp( R )`, where R is a homalg ring. If such an operation were available, I would be happy to use it :)

Reply:
True, would be nice to have such mechanism, to give a ring a proper LaTeX-name, as already done for entries and matrices. Such a name could also be used for other purposes.

Reply:
Also one thing, the blue color does not stop where it should be. Try for example the code:

```
ZZ := HomalgRingOfIntegers();
ZZrows := CategoryOfRows( ZZ );
Print( LaTeXOutput( IdentityMorphism( IdentityMorphism( 3/ZZrows ) / FreydCategory( ZZrows ) ) ) );
```

Reply:
You can use `{\color{blue} math-code}` instead of `\color{blue}{math-code}`

Reply:
Thanks, I'll do that. It's curious that it works fine within a jupyter notebook for me.

Reply:
Fixed

Reply:
I think the loaded packages and their order might interfere with each other and cause such effects.

Reply:
You mean LaTeX packages? Yes, maybe.

Reply:
Of course :)